module/VuFind/src/VuFind/Controller/AbstractBase.php

```diff
@@ -655,6 +655,17 @@ protected function commentsEnabled()
         return $check->getCommentSetting() !== 'disabled';
     }
 
+    /**
+     * Are ratings enabled?
+     *
+     * @return bool
+     */
+    protected function ratingsEnabled()
```

Comment:
Probably worth putting a return type on this (I try to type anything new that I add):

```suggestion
    protected function ratingsEnabled(): bool
```